edk2toollib/uefi/edk2/path_utilities_test.py

```diff
@@ -607,9 +607,10 @@ def test_get_relative_path_when_packages_path_list_contains_substrings(self):
         self.assertEqual(pathobj.GetEdk2RelativePathFromAbsolutePath(p), f"{ws_p_name}/module2/X64/TestFile2.c")
 
     def test_get_relative_path_when_package_path_inside_package(self):
```

Comment:
Fair enough, when I made the original (broken) code, I didn't think to glob downwards to see if a package was actually in the package path. I just made the assumption that if someone was declaring a directory a package_path, it will have a package in it.

In this case, I would like to keep the test as is, but build an additional test that adds a package to the package path. so we can get some test coverage on the assert. 

Would you be willing to add that? If not, I can do it in another PR.

Reply:
I just didn't realize it wasn't covered yet; it should definitely be covered as part of these changes.

I missed the instantiation of Edk2Path in the assert in that test case and I will add it back.